tests/e2e/pact/services/update-prescriptions.ts

```diff
@@ -29,10 +30,13 @@ const x509CertificatePath = process.env.SIGNING_CERT_PATH
 
 const isProd = process.env.APIGEE_ENVIRONMENT === "prod"
 
+let amendBundleIdentifier = null
```

Comment:
don't think we need this to be a top level variable, could sit inside the `updatePrescriptions` function I tihnk